crates/shim-protos/src/shim/mod.rs

```diff
@@ -13,3 +13,6 @@ pub mod shim_ttrpc_async;
 pub(crate) mod empty;
 pub(crate) mod mount;
 pub(crate) mod task;
+
+mod fieldpath;
+mod gogo;
```

Comment:
public or not?

Reply:
I plan to remove the generated code from the repo, so I don't have strong preference here.